pagopa/io-app
# fix(IDPay): [IODPAY-124] Fix scroll behavior in instrument and IBAN enrollment screen

## Short description
This PR fixes the scroll behavior in `InstrumentsEnrollmentScreen` and `IbanEnrollmentScreen`.

| Before 	| After 	|
|---	|---	|
| <video src="https://user-images.githubusercontent.com/6160324/228606049-7f49bea2-2ba0-47b7-b19f-c367b1d758f4.mp4" /> | <video src="https://user-images.githubusercontent.com/6160324/228605119-875a045f-63f8-45b5-9d90-f21db2d7b0c8.mp4" /> 	|

## List of changes proposed in this pull request
- [ts/features/idpay/initiative/configuration/screens/InstrumentsEnrollmentScreen.tsx](https://github.com/pagopa/io-app/pull/4497/files#diff-e57ad3d99b7dbdc319850166ea2e2c79d356a9c9ebfa3606c0ed51d139289157)
  - Moved `ScrollView`
  - Added styles
- [ts/features/idpay/initiative/configuration/screens/IbanEnrollmentScreen.tsx](https://github.com/pagopa/io-app/pull/4497/files#diff-a48aa011cee1d767d43c266ca5f3006ab59ff7aa05d1eee291f2866fbfb39085)
  - Moved `ScrollView`
  - Added styles

## How to test
Open the instrument configuration screen of an IDPay initiative and check the correct scroll behavior.
